syntax = "proto3";

package torch2trt;

message InterpolatePluginMsg {
  repeated int64 size = 1;
  string mode = 2;
  bool align_corners = 3;
}